### ISSUE TITLE
SignIn Missing Heading Hierarchy

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#788]
There are no headings whatsoever in the main content of the Sign In page.  Screen reader users are not presented with any way to navigate through the content other than from start to finish using the arrow keys, and other than the text, “Login using”, there is no indication that this is the Sign In page at all. 
 
Suggested Solution 
 
Add a level 1 heading with the text “Sign In”, between the logo and “Login using” text.  Then, mark up the “Login using” text as a heading at level 2. 

Proposed Code Sample 
 
File: src/components/SignIn/index.js 

`<h1 …>Sign In</h1> <h2 id="signInServices" className={signIn.text}>Login using</h2> `

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/ef3e079bbecde679ac50f27855190cc30a9c30fc.
